devicon.json

```
@@ -355,7 +355,7 @@
                 "line-wordmark"
             ]
         },
-        "color": "#303284",
+        "color": "#ab0001",
```

Same color as the main fill in the `original-wordmark` SVG
```suggestion
        "color": "#aa0001",
```

---

Sure. Done